### ISSUE TITLE
tags must in the same order as entered while typing the rant

### ISSUE THREAD

**Issue by chhanganivarun** [devRant/devRant#393]
Correct me of I'm wrong, Shouldn't it be better to have tags in the same order as typed?
@DFoxinator

**Comment by joas8211** [comment]
I agree on some level. It does make sence when writing and is always the assumption when writing a first rant (guidance here needed).

**Comment by joas8211** [comment]
But in the other end of the spectrum, the sorting guarantees that tags are used properly: not for long messages.
